docs/getting-started/README.md

```diff
@@ -0,0 +1,74 @@
+# Getting started
+
+## System requirements
+
+| Software | Minimum Version |
+| ------------- | ------------- |
+| [Node.js](https://nodejs.org/en/) | 12.x+ |
+| [MongoDB](https://docs.mongodb.com/manual/administration/install-community/)  | 3.6+ |
+| [ImageMagick](https://imagemagick.org/script/download.php#macosx) (optional) | Any |
+
+See the links above to install each. To check that these are installed in your environment, try the following commands:
```

Comment:
Up for debate, but I do remember being in node nightmares because installed through this instead of `brew`, which I do think we should encourage. I realize that this would be macOS heavy, but I think we should nudge them. I install almost everything this way.

Reply:
eh, research shows that no one else really cares. let it be.

Reply:
![image](https://user-images.githubusercontent.com/32125/111994681-68cc4880-8aee-11eb-8886-f1fe03038766.png)

Reply:
But seriously, it's fine.

Reply:
In the case of MongoDB, they do include primarily recommend using Homebrew, for what it's worth.